packages/slice-machine/server/src/api/common/generate.ts

```diff
@@ -20,14 +22,21 @@ export function generate(
     );
 
     components.forEach((c) => {
-      if (
-        !Files.exists(
```

Comment:
trying to think of what this was there for :/

Reply:
It used to check if we had existing mocks. I changed the logic to check 2 things:
- If we have existing mocks
- if the mocks are valid (using SharedSliceContent validator)

If one of these case is false, we regenerate